### PR TITLE
feat: adding new description for instructions and remove features

### PR DESCRIPTION
**Pull request by johnvente**
### Description

Description for the instructions updated and features list for the instructions removed.

BEFORE            |  NOW
:-------------------------:|:-------------------------:
  |  

#### Edit view
edit-view-after

### How to test it

- Check this branch`jv/feat-remove-feature-instructions` in the XBlock files manager

1. use node 18
 ```
nvm use
```

2. install dependencies
```
yarn install 
```
3 run to see any change in the react code 
```
yarn dev 

```

**Merge checklist:**
Check off if complete *or* not applicable:
- [x] Version bumped
- [x] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Fixup commits are squashed away
- [ ] Unit tests added/updated
- [ ] Manual testing instructions provided
- [ ] Noted any: Concerns, dependencies, migration issues, deadlines, tickets

### CODE REVIEW

**Review by Ian2012 — APPROVED** [review]

**Review by mariajgrimaldi — APPROVED** [review]

File: react-app/constants.js [hunk 1]
```diff
@@ -1,18 +1,9 @@
-export const basicFeatures = ['View a list of folders and files', 'Navigate through folders', 'Download files', 'Preview files', 'Search files by name', 'Select files and/or folders'];
+export const basicDescriptionInstructions = 'Use this component to upload and organize files that students can access directly in the unit.';
+export const advancedDescriptionInstructions = `${basicDescriptionInstructions} Any files you upload will be added to the "File and uploads" section. Extracting files from the "unpublished" folder leaves them visible to students, returning them to the folder hides them.`;
```
Comment (BryanttV) [inline react-app/constants.js:2]: In the latest versions of the platform, it is not called "Files & Uploads" but only "Files". We could add those two.

```suggestion
export const advancedDescriptionInstructions = `${basicDescriptionInstructions} Any files you upload will be added to the "File & Uploads" or "Files" section. Extracting files from the "Unpublished" folder leaves them visible to students, returning them to the folder hides them.`;
```

File: react-app/constants.js [hunk 2]
```diff
@@ -1,18 +1,9 @@
-export const basicFeatures = ['View a list of folders and files', 'Navigate through folders', 'Download files', 'Preview files', 'Search files by name', 'Select files and/or folders'];
+export const basicDescriptionInstructions = 'Use this component to upload and organize files that students can access directly in the unit.';
```
Comment (BryanttV) [inline react-app/constants.js:1]: This description is visible from the LMS, but students cannot do file uploads. I think we should modify that description.

Comment (johnvente) [reply react-app/constants.js]: I've changed it to this

lms-view

File: react-app/constants.js (hunk 1 above)
Comment (johnvente) [reply react-app/constants.js]: Done!
edit-view-test

**Review by BryanttV — APPROVED** [review]

**Pull request merged**